### PR TITLE
Events main page

### LINKED ISSUE #17: Event View Page
Page Url: `/events`
Module: Events

Components: 
- [ ] Tab Bar
- [ ] Float Button

Usage: View events list

- [ ] Development
- [ ] Unit Test

### PR DESCRIPTION
**Pull request by junzai97**
resolve #17

### CODE REVIEW

File: src/main/webapp/app/entities/event/event.tsx [hunk 1]
```diff
@@ -57,143 +69,64 @@ export class Event extends React.Component<IEventProps, IEventState> {
     this.props.getEntities(activePage - 1, itemsPerPage, `${sort},${order}`);
   };
 
+  toggleModal = eventId => {
```
Comment (HowardYaw) [inline src/main/webapp/app/entities/event/event.tsx]: method name toggle but you are doing open modal

File: src/main/webapp/app/routes.tsx [hunk 2]
```diff
@@ -29,6 +31,8 @@ const Routes = () => (
       <ErrorBoundaryRoute exact path="/auth/email/login" component={AuthEmailLogin} />
       <ErrorBoundaryRoute exact path="/auth/email/register" component={AuthEmailRegister} />
       <ErrorBoundaryRoute exact path="/auth/email/reset" component={AuthEmailReset} />
+      <ErrorBoundaryRoute exact path="/events/upcoming" component={Events} />
+      <ErrorBoundaryRoute exact path="/events/:id/details" component={EventDetails} />
```
Comment (HowardYaw) [inline src/main/webapp/app/routes.tsx]: There is a path defined in index.tsx of entity/event directory i think

**Review by HowardYaw — CHANGES_REQUESTED** [review]

File: src/main/webapp/app/entities/event/event.tsx [hunk 3]
```diff
@@ -14,6 +14,7 @@ import {
   JhiItemCount
 } from 'react-jhipster';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEllipsisH } from '@fortawesome/free-solid-svg-icons/faEllipsisH';
```
Comment (HowardYaw) [inline src/main/webapp/app/entities/event/event.tsx]: this one should be import in icon-loader.ts

File: src/main/webapp/app/routes.tsx (hunk 2 above)
Comment (junzai97) [reply src/main/webapp/app/routes.tsx]: Resolve in 19e9cf4e40e2cb39d14dcee1fa6d29764e323827

File: src/main/webapp/app/entities/event/event.tsx (hunk 3 above)
Comment (junzai97) [reply src/main/webapp/app/entities/event/event.tsx]: Resolved in 7f7222ecea72e3c604f27767af7c3e4f433c1c1d

File: src/main/webapp/app/entities/event/event.tsx (hunk 1 above)
Comment (junzai97) [reply src/main/webapp/app/entities/event/event.tsx]: Resolve in 7f7222ecea72e3c604f27767af7c3e4f433c1c1d

**Review by HowardYaw — APPROVED** [review]

**Review by LUXIANZE — APPROVED** [review]

**Pull request merged**